src/SdfEntityCreator.cc

```diff
@@ -465,14 +499,26 @@ Entity SdfEntityCreator::CreateEntities(const sdf::Joint *_joint)
 
   if (_joint->Axis(0))
   {
+    auto resolvedAxis = ResolveJointAxis(_joint->Axis(0));
+    if (!resolvedAxis)
+    {
+      return kNullEntity;
```

Comment:
Can you add an error message here? The error message in `ResolveJointAxis` doesn't have enough context about the model and joint associated with the error.

Reply:
Added, but the model entity isn't in this method, so the error just mentions the joint.